kinetic/distribution.yaml

```diff
@@ -3677,16 +3677,16 @@ repositories:
       - iirob_filters
       type: git
       url: https://github.com/KITrobotics/force_torque_sensor.git
-      version: kinetic-devel
+      version: melodic
```

Comment:
It's a bit odd to use a "newer" distribution branch name for an older distribution.  Usually if a branch can be used for both, the name of the "older" distribution is used.  That being said, this is allowed, so I'm just double-checking that this is what you meant to do.

Reply:
Hi, @clalancette Thanks for Your comment. I am switching everything to melodic and because of that, I decided to go with melodic for future compatibility. Sorry for the confusion, but this is OK.